storage/src/main/java/org/apache/uniffle/storage/common/DefaultStorageMediaProvider.java

```diff
@@ -61,8 +61,9 @@ public StorageMedia getStorageMediaFor(String baseDir) {
       try {
         File baseFile = new File(baseDir);
         FileStore store = Files.getFileStore(baseFile.toPath());
-        String mountPoint = store.name(); // mountPoint would be /dev/sda1 or /dev/vda1, etc.
-        String deviceName = mountPoint.substring(mountPoint.lastIndexOf(File.separator));
+        String mountPoint = store.name(); // mountPoint would be /dev/sda1, /dev/vda1, rootfs, etc.
+        int separatorIndex = mountPoint.lastIndexOf(File.separator);
+        String deviceName = separatorIndex > -1 ? mountPoint.substring(separatorIndex) : mountPoint;
```

Comment:
Let's turn line 62 to 67 to a method and add a UT for it?
`String deviceName = getDeviceName(baseDir)`;

Reply:
> Let's turn line 62 to 67 to a method and add a UT for it? `String deviceName = getDeviceName(baseDir)`;

`mountPoint` is different for different environments. I have no good idea to do this. May be we can turn line 64 to 67 to a method and add a UT for it.

Reply:
No suitable construction method found for `FileStore`, so I just turn line 65 to 67 to a method.